portfolio/src/main/webapp/style.css

```diff
@@ -77,3 +77,12 @@ p, li {
     font-family: 'Lato', sans-serif;
     font-size: 1.5em;
 }
+
+textarea, input {
+  font-family: inherit;
+  font-size: inherit;
+}
+
+textarea {
+  min-width: 50%;
+}
```

Comment:
Avoid applying specific styles to the general tag and prefer using a more specific class instead. Because in the future if you add different textareas that might want to do width 25% then you'd have to change it.